src/MadNLP.jl

```diff
@@ -15,7 +16,9 @@ export MadNLPSolver, MadNLPOptions, UmfpackSolver, LDLSolver, CHOLMODSolver, Lap
 import LDLFactorizations
 
 # Version info
-version() = parsefile(joinpath(@__DIR__,"..","Project.toml"))["version"]
+# version() = parsefile(joinpath(@__DIR__,"..","Project.toml"))["version"]
+const PROJECT_TOML = @path joinpath(@__DIR__,"..","Project.toml")
```

Comment:
Does the following work with PackageCompiler.jl? 
```julia
version() = packageversion(@__MODULE__)
```
Using that we don't have to parse the toml file and won't have to depend on RelocatableFolders.jl. 

This requires Julia >= 1.9

Reply:
I can try but I don't find this `packageversion` method is custom or part of some Package?

Reply:
My apologies, I meant `pkgversion` instead of `packageversion`. The function is part of `Base`.